test/test-helper.js

```diff
@@ -41,4 +42,8 @@ function dropTable (conn, name) {
   return deferred.promise
 }
 
```

Comment:
One (weird) trick (discovered by a teenager) I've been using in the user-acl-two and static-analysis-service tests has been to use `beforeEach` functionality in order to set up a single PG connection to hand to ormnomnom's `setConnection` machinery, so each unit test runs in a `BEGIN / ROLLBACK` pair (that way individual unit tests cannot share database state.)